packages/type-utils/src/getConstrainedTypeAtLocation.ts

```diff
@@ -4,8 +4,18 @@ import type {
 } from '@typescript-eslint/typescript-estree';
 import type * as ts from 'typescript';
 
+import * as tsApiUtils from 'ts-api-utils';
```

Comment:
[Style] Since this is its name everywhere else:

```suggestion
import * as tsutils from 'ts-api-utils';
```